lib/videoClient.ts

```diff
@@ -192,64 +221,68 @@ const createMeeting = async (credential, calEvent: CalendarEvent): Promise<any>
   };
 
   const organizerMail = new VideoEventOrganizerMail(calEvent, uid, videoCallData);
-  const attendeeMail = new VideoEventAttendeeMail(calEvent, uid, videoCallData);
+  const attendeeMail = new VideoEventAttendeeMail(calEvent, uid, videoCallData, emailTemplates);
   try {
     await organizerMail.sendEmail();
   } catch (e) {
-    console.error("organizerMail.sendEmail failed", e)
+    console.error("organizerMail.sendEmail failed", e);
   }
 
   if (!creationResult || !creationResult.disableConfirmationEmail) {
     try {
       await attendeeMail.sendEmail();
     } catch (e) {
-      console.error("attendeeMail.sendEmail failed", e)
+      console.error("attendeeMail.sendEmail failed", e);
     }
   }
 
   return {
     uid,
-    createdEvent: creationResult
+    createdEvent: creationResult,
   };
 };
 
-const updateMeeting = async (credential, uidToUpdate: String, calEvent: CalendarEvent): Promise<any> => {
+const updateMeeting = async (credential, uidToUpdate: string, calEvent: CalendarEvent): Promise<any> => {
   const newUid: string = translator.fromUUID(uuidv5(JSON.stringify(calEvent), uuidv5.URL));
 
   if (!credential) {
-    throw new Error("Credentials must be set! Video platforms are optional, so this method shouldn't even be called when no video credentials are set.");
+    throw new Error(
+      "Credentials must be set! Video platforms are optional, so this method shouldn't even be called when no video credentials are set."
+    );
   }
 
-  const updateResult = credential ? await videoIntegrations([credential])[0].updateMeeting(uidToUpdate, calEvent) : null;
+  const updateResult = credential
+    ? await videoIntegrations([credential])[0].updateMeeting(uidToUpdate, calEvent)
+    : null;
 
-  const organizerMail = new EventOrganizerRescheduledMail(calEvent, newUid);
-  const attendeeMail = new EventAttendeeRescheduledMail(calEvent, newUid);
+  const organizerMail = new EventOrganizerRescheduledMail(calEvent, newUid, []);
+  const attendeeMail = new EventAttendeeRescheduledMail(calEvent, newUid, []);
   try {
     await organizerMail.sendEmail();
   } catch (e) {
-    console.error("organizerMail.sendEmail failed", e)
+    console.error("organizerMail.sendEmail failed", e);
   }
 
   if (!updateResult || !updateResult.disableConfirmationEmail) {
     try {
       await attendeeMail.sendEmail();
     } catch (e) {
-      console.error("attendeeMail.sendEmail failed", e)
+      console.error("attendeeMail.sendEmail failed", e);
     }
   }
 
   return {
     uid: newUid,
-    updatedEvent: updateResult
+    updatedEvent: updateResult,
   };
 };
 
-const deleteMeeting = (credential, uid: String): Promise<any> => {
+const deleteMeeting = (credential, uid: string): Promise<any> => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689304)
```suggestion
const deleteMeeting = (credential, uid: string): Promise<unknown> => {
```